fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchQuantityTest.java

```diff
@@ -170,163 +193,201 @@ public void testSearchQuantity_Quantity_NoDisplayUnit() throws Exception {
     public void testSearchQuantity_Quantity_NoCode() throws Exception {
         assertSearchReturnsSavedResource("Quantity-noCode", "1||eq");
     }
-    
+
     @Test
     public void testSearchQuantity_Quantity_NoCodeOrUnit() throws Exception {
         // spec isn't clear about whether quantities with no unit should be indexed
         // but since we require the unit while searching, it doesn't really matter
         assertSearchDoesntReturnSavedResource("Quantity-noCodeOrUnit", "1||eq");
     }
-    
+
     /***
-     * FHIR Server does not yet use quantity comparator to calculate search results. *
+     * FHIR Server does not yet use quantity comparator to calculate search results.
      *********************************************************************************/
     // Quantity search is of the form <prefix><number>|<unit_system>|<unit>.
     // We use custom units to mark the quantity comparators so we can scope our searches in the tests.
-    
+
     @Test
     public void testSearchQuantity_Quantity_LessThan() throws Exception {
-        // Later versions of the spec indicate that there is an implicit precision 
-        // of .5 of the next least significant digit.  We don't support that now, but 
-        // lets use numbers far enough away that it won't matter.
-//        assertSearchReturnsSavedResource("Quantity-lessThan", "2||lt");
+        // For more details, please see Quantity search does not consider the quantity comparator field #493 https://github.com/IBM/FHIR/issues/493
+        // assertSearchReturnsSavedResource("Quantity-lessThan", "2||lt");
         assertSearchDoesntReturnSavedResource("Quantity-lessThan", "4||lt");
-        
+
+        // For more details, please see Quantity search does not consider the quantity comparator field #493 https://github.com/IBM/FHIR/issues/493
         // With implicit ranges, 3 (+/-0.5) actually might be < 3
```

Comment:
This comment should be removed if we remove the test it was commenting, which was:
`assertSearchDoesntReturnSavedResource("Quantity-lessThan", "3||lt");`

However, I don't think this test should have been removed (see next comment)